grpc/grpc.go

```diff
@@ -80,3 +86,57 @@ func (mi *ModuleInstance) Exports() modules.Exports {
 		Named: mi.exports,
 	}
 }
+
+// stream returns a new stream object
+func (mi *ModuleInstance) stream(c goja.ConstructorCall) *goja.Object {
+	rt := mi.vu.Runtime()
+
+	client, ok := c.Argument(0).ToObject(rt).Export().(*Client)
```

Comment:
Not certain but I am pretty sure this will panic if there are 0 arguments provided :thinking: 

We can fix this in a separate PR

Reply:
Nah, it will be a TypeError, but I'll fix it in the following PRs where I'll work more with this constructor